playbooks/roles/wordpress/tasks/nginx.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: open HTTPS port on the firewall
+  ufw:
+    rule: allow
+    port: "{{ item }}"
+    proto: tcp
+  with_items:
+    - 443
+
+- name: mount /var/www
+  mount: name=/var/www src=/dev/vdb1 fstype=ext4 state=mounted
+
+- name: chown www-data /var/www
+  file: path=/var/www mode=750 owner=www-data group=www-data state=directory
+
+- name: mkdir /var/www/log
+  file: path=/var/www/log mode=750 state=directory
+
+- name: Ensure directory for config files exists
+  file:
+    path: /etc/nginx/sites-available/wordpress
+    state: directory
+
+- name: Copy nginx global configuration
+  synchronize: src="global/" dest=/etc/nginx/global checksum=yes delete=yes owner=no group=no
+
+- name: Add landing site nginx configuration
```

Comment:
@Agrendalath, nit: might be a good idea to combine this with the next 2 tasks by using placeholders and the `with_items`.

Reply:
Done.